src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1734,7 +1841,7 @@ private bool DoMatchWorker(string operandString, MatchInfo matchInfo, out MatchI
                 {
```

Comment:
This old 'StringComparison compareOption = ...' code can be removed.

Reply:
Done.